app/controllers/posts_controller.rb

```diff
@@ -1,36 +1,29 @@
 class PostsController < ApplicationController
   layout 'standard'
-
   def index
     @user = User.find(params[:user_id])
     @posts = Post.where(author_id: params[:user_id]).order(id: :asc)
```

Comment:
- [ **OPTIONAL BUT RECOMMENDED**] Please remove **line 5** or refactor your code as shown below to avoid sending unnecessary queries to your database, as you are now selecting the posts as requested by the previous.. :point_down: 

```
   @user = User.find(params[:user_id])
   @posts = @user.posts.includes(:comments).order(id: :asc).paginate(page: params[:page], per_page: 5)
```

### screenshot 
![Screenshot from 2023-11-08 17-46-59](https://github.com/JoyTaribagshaw/blog-app/assets/89228743/1bd2eab4-dcc1-4e02-a057-03f7cae78214)